### ISSUE TITLE
Exception in thread "main" java.long.RuntimeException":

### ISSUE THREAD

**Issue by cardaminexhz** [secure-software-engineering/SuSi#9]
Hi,
I have tried to use SuSi to generate Android sources and sinks, so that I can use them as input in FlowDroid to generate CFG. But now I encountered this problem when I tried to run SuSi by input this in command line(windows):
`java -cp lib/weka.jar;soot-trunk.jar;soot-infoflow.jar;soot-infoflow-android.jar;SuSi.jar de.ecspride.sourcesinkfinder.SourceSinkFinder android-10.jar permissionMethodWithLabel.pscout out.pscout`
Then, I got this error.
![1](https://cloud.githubusercontent.com/assets/9798380/14984940/d7087be8-1177-11e6-82a1-fabb74d94e65.png)

  I get 'andorid-10.jar' from Sable/android-platforms(I have also tried some other android-xx.jars, e.g. 9, 11, 13, 15, 17, still got the same error), and get other required jars from  the place where  StevenArzt mentioned in issue #3 .
![image](https://cloud.githubusercontent.com/assets/9798380/14984735/d21f120a-1176-11e6-9933-c336530f04e1.png)

So, may somebody offer me some help?

**Comment by StevenArzt** [comment]
Your android-10.jar file looks a bit too small with only 44 KB. It should be roughly 8 MB of size. Did you really use this link? https://github.com/Sable/android-platforms/blob/master/android-10/android.jar?raw=true

**Comment by cardaminexhz** [comment]
Thank you for pointing this. I should be more careful.

But then I got another problem. I operated in branch develop, use the right Android.jar.
![qq 20160504102907](https://cloud.githubusercontent.com/assets/9798380/15003856/f5f585d0-11e2-11e6-8394-e8d40ee86ff6.png)

The problem is shown as followed
![qq 20160504103009](https://cloud.githubusercontent.com/assets/9798380/15003871/20098ab0-11e3-11e6-93db-fa9fc12e2acf.png)

**Comment by StevenArzt** [comment]
Are you using our original input file with the annotations or did you modify it? Apparently, the machine learner is complaining about inconsistent classifications in that file.

**Comment by cardaminexhz** [comment]
Yes, I used the original input file without modification.  I just copyed soot-trunk.jar;soot-infoflow.jar;soot-infoflow-android.jar;SuSi.jar and Android.jar to the same directory.
Then, I execuated the command.
I've tried several times, but just got the same problem.
Thanks for your time.

**Comment by lizheng5916** [comment]
I also meet  the problem ,have you solve the exception  ?

**Comment by cardaminexhz** [comment]
Sorry, I didn't.

**Comment by lizheng5916** [comment]
windows platform cause the problem?  who knows?

**Comment by Zhongen** [comment]
Hi, where I can get SuSi.jar file?

**Comment by freefire4547** [comment]
He has provided link for nightly build but its not working. I imported project and built using IDE.
